src/basic/embeds/position_embed.jl

```diff
@@ -18,9 +18,9 @@ get_value(e::PositionEmbedding, name::Symbol, xs::NamedTuple) = e(first(xs))
 
 function PE(size, pos, i::Int)
     if rem(i, 2) == 0
-        sin(pos/1e4^(i/size))
+        sin((pos-1)/1e4^((i-2)/size))
     else
-        cos(pos/1e4^((i-1)/size))
+        cos((pos-1)/1e4^((i-1)/size))
```

Comment:
```julia
if rem(i, 2) == 1
    sin((pos-1)/1e4^((i-1)/size))
else
    cos((pos-1)/1e4^((i-2)/size))
end
```

Let's switch the order with something like this, so the output matrix can be exactly the same

Reply:
ok, good idea. I updated the order in the revised PR.